src/common/chat-participants/powerpages/PowerPagesChatParticipantUtils.ts

```diff
@@ -57,6 +57,17 @@ export function isEntityInSupportedList(entity: string): boolean {
     return SUPPORTED_ENTITIES.includes(entity);
 }
 
+export function createAndReferenceLocation(activeFileUri: vscode.Uri, startLine: number, endLine: number): vscode.Location {
+
+    const positionStart = new vscode.Position(startLine, 0);
+    const positionEnd = new vscode.Position(endLine, 0);
+
```

Comment:
nit: we can club all these variables together in definition at start of function

Reply:
Thanks, done.